development/nautobot_config.py

```diff
@@ -19,7 +30,9 @@
     }
 }
 
-DEBUG = True
+#
+# Logging
+#
 
 LOG_LEVEL = "DEBUG" if DEBUG else "INFO"
```

Comment:
This is busted now since we're not setting `DEBUG = True` until after we check its value here to influence the `LOG_LEVEL` setting.